app/controllers/concerns/avatarable.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Avatarable
+  extend ActiveSupport::Concern
+
+  def user_avatar(user)
+    return url_for(user.avatar) if user.avatar.attached?
+
+    ActionController::Base.helpers.image_url('default-avatar.png')
```

Comment:
Can we drop `ActionController::Base.helpers.` now? and just do `image_url`?

Reply:
We can't.
Althought it seems view_context (from ActionView) should/could be used instead of ActionController.

`view_context.image_url('default-avatar.png')  `

Reply:
After some research, I think view_context makes more sense - and is cleaner. Do you agree?

Reply:
Yeah